mistlib/core.py

```diff
@@ -399,6 +399,80 @@ def append_file(input_filename, output_filename):
                 output_file.write(input_content)
     
             return
+    
+    def write_additivefoam_input(self, file):
+        specific_heat_1 = None
+        specific_heat_2 = None
+        # Reference temperature based on a property from self.properties
+        comment_block = """/*---------------------------------------------------------------------------
+     AdditiveFOAM template input file (compatible with 1.0, OpenFOAM 10)
+
+                      Created for simulation with Myna
+  ---------------------------------------------------------------------------*/
+  FoamFile
+{
+    version     2.0;
+    format      ascii;
+    class       dictionary;
+    object      transportProperties;
+}
+
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+"""
+
+        reference_temperature = self.properties["solidus_eutectic_temperature"].value
+        p=(self.properties["thermal_conductivity_solid"].value_laurent_poly)
+        labeled_values = {}
+        for idx, pair in enumerate(p, 1):
+            label = f"thermal_cond_S{idx}" 
+            first_value = pair[0]
+            labeled_values[label] = first_value
+       
+        a = []
```

Comment:
I strongly recommend using descriptive names whenever possible. Using something like `f` for a file or `p` for a property can be ok since the letter stands for something, but even that is best used when all uses are in close proximity.

```suggestion
        thermal_conductivity_coeffs = []
```